docs/execution-providers/Azure-ExecutionProvider.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Azure
+description: Instructions to infer an ONNX model remotely with an Azure endpoint
+parent: Execution Providers
+nav_order: 12
+redirect_from: /docs/reference/execution-providers/Azure-ExecutionProvider
```

Comment:
I think this line is not needed since this file never existed at previous location

Reply:
Removed.